### ISSUE TITLE
unused functions and classes

### ISSUE THREAD

**Issue by lionelkusch** [mind-inria/hidimstat#42]
In the file stat_tools.py, the functions two_sided_pval_from_cb, and two_sided_pval_from_pval are not used except for testing. 

In the file, Dnn_learner.py, the class DnnLearner is never used. Perhaps we need to add an example of it. 

In the file, RandomForestModified.py, the classes RandomForestClassifierModified and  RandomForestRegressorModified are not used. Perhaps we need to add an example of it.

**Comment by lionelkusch** [comment]
It seems that the file Dnn_learner.py and  RandomForestModified.py are associated with the example hidimstat/examples_not_exhibited/plot_residuals_sampling.py but it is missing the definition of the class BlockBasedImportance. 
I find this definition here: https://github.com/achamma723/Variable_Importance/tree/main/BBI_package/src/BBI

It seems that there is some desire to include the package BBI in this library, but it is actually half done. 
@bthirion, do we finish including this library in the Hidimstat, or do we remove the file and function link from it?

**Comment by bthirion** [comment]
Good question. @achamma723 @jpaillard I would like to have your input on this: are these examples deprecated ? Or is the main plan to integrate it into Hidmistat ? Please refresh me.

**Comment by jpaillard** [comment]
- DNNLearner: In my opinion one key feature of the CPI / LOCO methods is to be model agnostic, I would suggest not to have models (like DNNLearner) which come with heavy dependencies (PyTorch) as part of the main library. 
This would not hinder the use of PyTorch models, one just needs to use sklearn compatible models like the wrappers from skorch. I actually have an example on MNIST that I could include. 

 - For ModifiedRF, I would let Ahmad answer.

**Comment by achamma723** [comment]
Hello, for Modified RF, it is the script representing the sampling case of the Random Forest. The actual default case is to build the new samples by the mean of the shuffled residuals. The alternative in the ModifiedRF is to sample from the same leaves of the current sample (test sample) that could conserve the conditional dependency across the neighboring samples.

**Comment by lionelkusch** [comment]
The unused functions of utils.py have been removed.
The file  Dnn_learner.py and RandomForestModified.py have been removed for the moment. 

There are still the functions in stat_tools.py, which need to be removed or used.

**Comment by bthirion** [comment]
Close as completed.
